### PR TITLE
Add a new database question

### PR DESCRIPTION
**Pull request by fangjian0423**
1. Add a new database question: Which database would you like to use?

Fow now, the todo app only support PostgreSQL, so there is only one option.

2. Update ms docs link

### CODE REVIEW

File: generators/createtodoapp/generator.mjs [hunk 1]
```diff
@@ -146,7 +152,7 @@ ${chalk.cyan(`    mvn clean package -DskipTests`)}
 ${chalk.cyan(`    java -jar web\\target\\azure-spring-apps-todo-web-0.0.1-SNAPSHOT.jar`)}
 
 ${chalk.magentaBright(`Deploy on Azure Spring Apps with monthly free grants:`)}
-${chalk.cyan(`    https://review.learn.microsoft.com/en-us/azure/spring-apps/quickstart-for-web-app?branch=pr-en-us-227725`)}
+${chalk.cyan(`    https://learn.microsoft.com/azure/spring-apps/quickstart-deploy-web-app`)}
```
Comment (stliu) [inline generators/createtodoapp/generator.mjs:155]: I think we should use a aka.ms short link here

File: generators/createtodoapp/generator.mjs [hunk 2]
```diff
@@ -34,6 +34,12 @@ export default class extends BaseGenerator {
             name: "packageName",
             message: "What is your default java package name?",
             default: "com.mycompany.myapp"
+          },
+          {
+            type: 'list',
+            name: 'prodDatabaseType',
+            message: 'Which database would you like to use?',
+            choices: ['PostgreSQL']
```
Comment (sinedied) [inline generators/createtodoapp/generator.mjs:42]:
```suggestion
            choices: ['postgresql']
```
Value should be in lowercase to match existing values by the jhipster generator

File: generators/createtodoapp/generator.mjs (hunk 1 above)
Comment (hangwan97) [reply generators/createtodoapp/generator.mjs:155]: https://shorturl.at/fjKU6

**Review by hangwan97 — APPROVED** [review]

File: generators/createtodoapp/generator.mjs (hunk 1 above)
Comment (fangjian0423) [reply generators/createtodoapp/generator.mjs]: https://aka.ms/asa/webapp-quickstart

**Review by stliu — APPROVED** [review]

**Pull request merged**